### ISSUE TITLE
Broken ENV values in Frontend

### ISSUE THREAD

**Issue by S-Nigel** [Amsterdam/openstad-kubernetes#35]
The values API and IMAGE_API_URL have nil values:
```
Environment:
      DEFAULT_HOST:            www.openstad.softwaredepartment.net
      APP_URL:                 https://www.openstad.softwaredepartment.net
      API:                     https://www.
      IMAGE_API_URL:           https://www.
```

Probably caused by a broken reference in the Helm template

**Comment by ToshKoevoets** [comment]
I've fixed this in PR

**Comment by S-Nigel** [comment]
Oh awesome! Thank you @ToshKoevoets 

Fixed in PR: #33 (Need review)
